prql/src/ast.rs

```diff
@@ -273,3 +303,9 @@ impl From<Item> for anyhow::Error {
         anyhow!("Failed to convert {item:?}")
     }
 }
+
+impl Dialect {
+    pub fn use_top(&self) -> bool {
+        matches!(self, Dialect::MsSql)
```

Comment:
Possibly in the future we want to have each Dialect be a Struct that we implement a `Trait` for — for the moment this is excellent.